packages/ui/src/components/BlockList.tsx

```diff
@@ -89,17 +97,18 @@ class _BlockList extends React.Component<Props, {}> {
             </tr>
           );
         }}
-        filterRow={(block: JsonBlock) => {
+        filterRow={(block: BlockResult) => {
           // fixme
```

Comment:
The `filterRow` function could enable user to filter the page by type of message, could we do it by using the endpoint from event store now?